mpas_analysis/ocean/ocean_modelvsobs.py

```diff
@@ -93,11 +93,11 @@ def ocn_modelvsobs(config, field):
 
     elif field.lower() == 'sst':
 
-        ds = xr.open_mfdataset(infiles, preprocess=lambda x: preprocess_mpas(x, yearoffset=yr_offset,
-		                    timeSeriesStats=True,
-		                    timestr='time_avg_daysSinceStartOfSim',
-		                    onlyvars=['time_avg_activeTracers_temperature'],
-		                    selvals={'nVertLevels':1}))
+        ds = xr.open_mfdataset(infiles, preprocess=lambda x: 
+                               preprocess_mpas(x, yearoffset=yr_offset,
+                                               timestr=['xtime_start', 'xtime_end'],
+		                               onlyvars=['time_avg_activeTracers_temperature'],
+		                               selvals={'nVertLevels':1}))
         ds = remove_repeated_time_index(ds)
         ds.rename({'time_avg_activeTracers_temperature':'mpasData'}, inplace = True)
 
```

Comment:
To make things backward compatible, it seems like we need a list of options for the timestr variable as well: did you want to handle this in the specific PR that will address #20?